### ISSUE TITLE
使用inference.py推理，但是没有任何图片保存下来。过了一段时间再运行相同的代码，就又有图片保存下来了。这种情况反复出现。

### ISSUE THREAD

**Issue by HHanTheKing** [tyxsspa/AnyText#157]
运行官方推理代码显示如下，但实际上我的SaveImages文件夹里没有任何图片：[/+]
[+]Loaded model config from [[/+]
[+]models_yaml/anytext_sd15.yaml][/+]
[+]Loaded state_dict from [/mnt/petrelfs/hantao.dispatch/.cache/modelscope/hub/damo/cv_anytext_text_generation_editing/anytext_v1.1.ckpt][/+]
[+]Original weights loaded![/+]
[+]Global seed set to 66273235[/+]
[+]Data shape for DDIM sampling is (2, 4, 64, 64), eta 0.0[/+]
[+]Running DDIM Sampling with 20 timesteps[/+]
[+]DDIM Sampler: 100%|████████████████████████████████████████████████████████████████████████████| 20/20 [00:02<00:00, 8.85it/s][/+]
[+]Prompt: photo of caramel macchiato coffee on the table, top-down perspective, with "Any" "Text" written on it using cream[/+]
[+]Done, result images are saved in: SaveImages[/+]
[+]Global seed set to 8943410[/+]
[+]Data shape for DDIM sampling is (2, 4, 64, 64), eta 0.0[/+]
[+]Running DDIM Sampling with 20 timesteps[/+]
[+]DDIM Sampler: 100%|████████████████████████████████████████████████████████████████████████████| 20/20 [00:02<00:00, 9.43it/s][/+]
[+]Prompt: A cake with colorful characters that reads "EVERYDAY"[/+]
[+]Done, result images are saved in: SaveImages
